baseplate/lib/message_queue.py

```diff
@@ -140,6 +218,11 @@ def queue_tool() -> None:
         help="if creating the queue, what to set the maximum message size to",
     )
     parser.add_argument("queue_name", help="the name of the queue to consume")
+    parser.add_argument(
+        "use_in_memory_queue",
+        default=False,
```

Comment:
If you wonna be a bit stricter here you can use `action="store_true"` [[docs](https://docs.python.org/3/library/argparse.html#action)] which will set the value to `True` if the flagg is pressent and `False` if abscent. Just means someone couldn't write `--use_in_memory_queue "dave"` and cause issues.